### PR TITLE
Make the application ASE-deployable

### PR DESCRIPTION
**Pull request by lgonczar**
### Change description ###

Make the service ASE-deployable, so that it can be deployed to demo.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -76,7 +76,9 @@ withPipeline(type, product, component) {
 
   enableDockerBuild()
   enableAksStagingDeployment()
-  disableLegacyDeployment()
+  onMaster() {
+    disableLegacyDeployment()
+  }
```
Comment (lgonczar) [inline Jenkinsfile_CNP]: @timja Is this safe enough?

**Comment by timja** [comment]
this is soo wrong :(
an application that has never been to the ASE should not go there

**Comment by lgonczar** [comment]
This app had to go to demo (and has been deployed there) in order to support testing with other services, while HMCTS demo isn't ready. We had to meet the deadline.

File: Jenkinsfile_CNP (hunk 1 above)
Comment (doncem) [reply Jenkinsfile_CNP]: there could be any other branch build/pr raised

**Comment by lgonczar** [comment]
> Can we do this only in demo branch ?

How about having the deployment unconditionally disabled, but the terraform changes merged to master? Otherwise, if demo is overwritten, we're going to lose these changes.

**Comment by nitinprabhu** [comment]
> > Can we do this only in demo branch ?
> 
> How about having the deployment unconditionally disabled, but the terraform changes merged to master? Otherwise, if demo is overwritten, we're going to lose these changes.

sounds good 👍

**Comment by lgonczar** [comment]
> Could you please run terraform fmt on the tf files ?

done

**Pull request merged**